docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,485 @@
-# Man Page
+# Man pages
+
+## Commands by functional group
+
+### Performance tool management commands
+
+- [pbench-clear-results](#pbench-clear-results)
+- [pbench-clear-tools](#pbench-clear-tools)
+- [pbench-list-tools](#pbench-list-tools)
+- [pbench-list-triggers](#pbench-list-triggers)
+- [pbench-register-tool](#pbench-register-tool)
+- [pbench-register-tool-set](#pbench-register-tool-set)
+- [pbench-register-tool-trigger](#pbench-register-tool-trigger)
+
+### Benchmark commands
+
+- [pbench-user-benchmark](#pbench-user-benchmark)
+
+### Upload to Pbench Server
+
+#### Pbench Server 0.69
+
+- [pbench-move-results](#pbench-move-results)
+- [pbench-copy-results](#pbench-copy-results)
+
+#### Pbench Server 1.0
+
+- [pbench-results-move](pbench-results-move)
```

Comment:
Oops: I just noticed that the link should be `(#pbench-results-move)` ... you lost the `#`.